src/components/Metrics/Metrics.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react'
+
+import { Props, State } from './Metrics.types'
+
+import './Metrics.css'
+import { SceneMetrics } from 'modules/scene/types'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+
+export default class Metrics extends React.PureComponent<Props, State> {
+  constructor(props: Props) {
+    super(props)
+    this.state = {
+      toggle: false
+    }
+  }
+
+  componentWillMount() {
+    document.addEventListener('click', this.handleClose)
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.handleClose)
+  }
+
+  handleToggle = () => {
+    this.setState({
+      toggle: !this.state.toggle
+    })
+  }
+
+  handleClose = () => {
+    this.setState({
+      toggle: false
+    })
+  }
+
+  handleClick = (event: React.MouseEvent<HTMLElement>) => {
+    event.nativeEvent.stopImmediatePropagation()
+  }
+
+  renderMetrics() {
+    return Object.keys(this.props.metrics).map(key => this.renderMetric(key as keyof SceneMetrics))
+  }
+
+  renderMetric(metric: keyof SceneMetrics) {
+    const { metrics, limits } = this.props
+    let classes = 'metric'
+    if (metrics[metric] > limits[metric]) {
+      classes += ' error'
+    }
+
+    return (
+      <div className={classes} key={metric}>
+        <div className="label">{metric}:</div>
```

Comment:
We might want to translate this, something like 
```suggestion
        <div className="label">{t(`metrics.${metric}`)}:</div>
```